docker/build_moab.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+source ${docker_env}
+
+if [ ${MOAB_VERSION} == "master" ] || [ ${MOAB_VERSION} == "develop" ]; then
+  branch=${MOAB_VERSION}
+else
+  branch=Version${MOAB_VERSION}
+fi
+
+rm -rf ${moab_build_dir}/bld ${moab_install_dir}
+mkdir -p ${moab_build_dir}/bld
+cd ${moab_build_dir}
+git clone https://bitbucket.org/fathomteam/moab -b ${branch}
```

Comment:
If you are using `git clone` for the sole purpose of building a binary, I recommend using the `--depth` flag to create a shallow clone of the repository [1].

Combining `--depth` with `-b` downloads only the most recent commit on the branch [2], and significantly decreases download latency.  E.g.:

```
git clone --depth 1 https://bitbucket.org/fathomteam/moab -b ${branch}
```

[1] https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt
[2] https://stackoverflow.com/a/21833871